code/game/gamemodes/events/holidays/Holidays.dm

```diff
@@ -43,7 +43,7 @@ var/global/Holiday = null
 
 		if(4) // Apr
 			if(DD != 1 && time2text(world.timeofday, "DDD") == "Sun") //no idea what would happen if april fools gets set twice so we check just in case
-				current_holidays += APRIL_FOOLS_DAY
+				current_holidays += list(APRIL_FOOLS_DAY, APRIL_NORMAL_DAY, APRIL_NORMAL_DAY, APRIL_NORMAL_DAY, APRIL_NORMAL_DAY, APRIL_NORMAL_DAY, APRIL_NORMAL_DAY, APRIL_NORMAL_DAY, APRIL_NORMAL_DAY, APRIL_NORMAL_DAY)
```

Comment:
Mulitple holidays can be on the same day. You'll have to use `pick()` here for it to only choose one of them.
```suggestion
				current_holidays += pick(APRIL_FOOLS_DAY, APRIL_NORMAL_DAY, APRIL_NORMAL_DAY, APRIL_NORMAL_DAY, APRIL_NORMAL_DAY, APRIL_NORMAL_DAY, APRIL_NORMAL_DAY, APRIL_NORMAL_DAY, APRIL_NORMAL_DAY, APRIL_NORMAL_DAY)
```

Reply:
The comment at the start of the file (which to be fair is over a decade old) and my testing found that it picks a holiday at random from the list.

Reply:
Weird. Fair enough!

Reply:
Yeah, it can only be one holiday at once. If there's a list, it picks at random. We learned this with the original weekend April fools PR.